src/Pages/seunghaoh/Main/Main.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "../../../Styles/reset.scss";
+import "../../../Styles/common.scss";
+import Feeds from "./Components/Feeds/Feeds";
+import Nav from "./Components/Nav/Nav";
+
+class Main extends React.Component {
+  render() {
+    return (
+      <div>
+        <Nav />
+        <Feeds />
+      </div>
```

Comment:
```suggestion
      <>
        <Nav />
        <Feeds />
      </>
```

- React Fragments 적용해볼 수 있겠네요!

Reply:
적용했습니다!